src/components/App.js

```diff
@@ -242,6 +242,13 @@ class App extends Component<Props, State> {
     }));
   }
 
+  triggerEditorPaneResize() {
+    const editorPane = window.document.querySelector(".editor-pane");
+    if (editorPane) {
+      editorPane.dispatchEvent(new Event("resize"));
```

Comment:
nit: also would it be clearer to dispatch `resizeonend` custom event here
```suggestion
      editorPane.dispatchEvent(new Event("resizeonend"));
```